Sources/KIF/Additions/UIView-KIFAdditions.m

```diff
@@ -931,6 +931,13 @@ - (CGPoint)tappablePointInRect:(CGRect)rect;
     UIView *hitView = nil;
     CGPoint tapPoint = CGPointZero;
     
+    // accessibilityActivationPoint
```

Comment:
Would be good to document here that it defaults to the center if no override is set.